scenes/game_elements/characters/npcs/elder/components/story_quest_starter.dialogue

```diff
@@ -8,6 +8,29 @@ if chosen_quest == null:
 	% StoryQuest Elder: Please stay. Our world will perish without your help!
 	% StoryQuest Elder: I hope to see you again. We really need you!
 	% StoryQuest Elder: The people of Threadbare are counting on you. I hope you change your mind!
+elif chosen_quest.resource_path in GameState.completed_quests:
+	# Custom dialogues for completed quests
+	if chosen_quest.resource_path == "res://scenes/quests/story_quests/el_juguete_perdido/quest.tres":
+		% StoryQuest Elder: Has rescatado a Apolo antes… y gracias a ti volvió sano y salvo.
+		% StoryQuest Elder: Pero ese chico siempre encuentra nuevas travesuras… quizá necesite tu guía otra vez.
+		% StoryQuest Elder: Si deseas volver al bosque, podrás revivir la búsqueda y perfeccionar tu camino.
+
+	elif chosen_quest.resource_path == "res://scenes/quests/story_quests/shjourney/quest.tres":
+		% StoryQuest Elder: Ya has enfrentado al bosque que devora recuerdos… pocos regresan de él.
+		% StoryQuest Elder: Si quieres regresar, las sendas del bosque estarán listas para probarte de nuevo.
+
+	elif chosen_quest.resource_path == "res://scenes/quests/story_quests/stella/quest.tres":
+		% StoryQuest Elder: Stella sigue mirando al cielo con esperanza gracias a ti.
+		% StoryQuest Elder: Si vuelves a acompañarla, podrás recorrer la historia desde una nueva luz.
+
+	elif chosen_quest.resource_path == "res://scenes/quests/story_quests/verso/quest.tres":
+		% StoryQuest Elder: Tu apoyo le devolvió la confianza a Nora cuando más lo necesitaba.
+		% StoryQuest Elder: Pero el conocimiento siempre pone nuevas pruebas a quienes regresan.
+		% StoryQuest Elder: Si decides intentarlo otra vez, podrás dominar cada paso del desafío.
+
+	else:
+		% StoryQuest Elder: Well done, traveler. Another story is restored.
+		% StoryQuest Elder: Hope grows stronger with every thread you weave.
```

Comment:
The `else:` fallback here (and in the other dialogue) is still giving a congratulations. But this is when the quest starts, after the storybook is displayed. It should at least say "Here you go again..." or similar.